apps/docs/docs/core/form-data.mdx

```diff
@@ -112,11 +112,11 @@ With Nest this is a pretty simple implementation, due to the extensible Decorato
 ```ts
 // nest
 @Controller()
-export class AppController implements ControllerShape {
+export class AppController implements NestControllerInterface<typeof c> {
   @Api(s.route.updateUserAvatar)
   @UseInterceptors(FileInterceptor('avatar'))
   async updateUserAvatar(
-    @ApiDecorator() { params: { id } }: RouteShape['updateUserAvatar'],
+    @TypedRequest() { params: { id } }: RequestShapes['updateUserAvatar'],
```

Comment:
I'm wondering if `TypedRequest` should just be named something to reflect the package name itself? 

For instance, importing `{ z } from 'zod'` feels super natural. So maybe something like, `TSRest` or `TSRestRoute` would look even better and allow people to easily distinguish when they're using the `ts-rest` library?

Reply:
So we would have:

```
import { TSRestRequest } from 'ts-rest/nest'

 async updateUserAvatar(
    @TSRestRequest() { params: { id } }: TSRestShapes['updateUserAvatar'],
```

Reply:
I agree fully. That was actually the initial name I came up with, but I felt like `TSRestRequest` was a little clunky to type and a bit of tongue-twister 😅

Let's think of something